src/components/error-modal.component.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {
+  InlineNotification,
+} from '@carbon/react';
+import { useTranslation } from 'react-i18next';
+
+const ErrorModal: React.FC<{error: Error}> = ({ error }) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+     <InlineNotification
+              style={{ minWidth: '100%' }}
+              kind="error"
+              lowContrast={true}
+              hideCloseButton={false}
+              title= {t('fieldErrorDescriptionTitle', 'Validation Error')}
+              subtitle={t('fieldErrorDescription', error.message)}
```

Comment:
In case we are passing a value and no string, we don't need the translation key, it creates an empty entry `fieldErrorDescription: ""`
```suggestion
              subtitle={error.message}
```